docs/files/horizontal-layout-template.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html lang="en"><!-- lang is necessary for screen reader compatibility -->
 <head>
-  <title>Horizontal - App Title</title>
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>Horizontal - App Title</title>
   <link rel="shortcut icon" href="images/favicon.png" />
 
   <!-- DO NOT BUNDLE: HelixUI Styles -->
```

Comment:
TODO: move adapter script to bottom of body (like `docs/_templates/partials/head.njk`).